lightning/src/routing/scoring.rs

```diff
@@ -494,7 +513,10 @@ pub struct ProbabilisticScoringParameters {
 	///
 	/// [`liquidity_penalty_amount_multiplier_msat`]: Self::liquidity_penalty_amount_multiplier_msat
 	pub historical_liquidity_penalty_amount_multiplier_msat: u64,
+}
 
+#[derive(Clone)]
+pub struct ProbabilisticScoringDecayParameters {
```

Comment:
`historical_no_updates_half_life` and `liquidity_offset_half_life` are moved into the new `ProbabilisticScoringDecayParameters`. the rest are kept in `ProbabilisticScoringFeeParameters` 

things are moved around, so it looks abit messy on the PR